### PR TITLE
Add API for visualizing crop margins

### PR DESCRIPTION
**Pull request by seanzhang15**

### CODE REVIEW

File: python/mujinplanningclient/planningclient.py [hunk 1]
```diff
@@ -430,6 +430,46 @@ def SetCameraTransform(self, pose=None, transform=None, distanceToFocus=0.0, tim
...
+    # 
+
+    def SetCropMarginVisible(self, name=None, visible=False, timeout=10, fireandforget=True, slaverequestid=None):
+        """
+        Sets the crop margin visualzation to be on or off
+
+        Args:
+            name: name of the kinbody
+            visible: boolean value setting the visibility of the crop margin visualization
+        """
```
Comment (felixvd) [inline python/mujinplanningclient/planningclient.py:444]:
```suggestion
        """Sets the crop margin visualzation to be on or off.

        Args:
            name (str, optional): Name of the KinBody.
            visible (bool): If False, the crop margin is not visualized.
        """
```
Ibid for the other one.

File: python/mujinplanningclient/planningclient.py [hunk 2]
```diff
@@ -430,6 +430,46 @@ def SetCameraTransform(self, pose=None, transform=None, distanceToFocus=0.0, tim
...
+        if name is None:
+            return
```
Comment (felixvd) [inline python/mujinplanningclient/planningclient.py:446]: Huh? Just remove the default for `name`.

Comment (felixvd) [reply python/mujinplanningclient/planningclient.py:446]: Also please follow the structure of the other commands (`viewercommand` first), this client will soon be generated

File: python/mujinplanningclient/planningclient.py (hunk 1 above)
Comment (seanzhang15) [reply python/mujinplanningclient/planningclient.py]: Improved docstring. Also combined the two apis into one.

**Comment by seanzhang15** [comment]
Instead of `viewercommand`, we'll be updating the `viewerparameters`. So we can use the old API `SetViewerFromParameters` and we no longer need the new API.

**Pull request closed without merging**